### PR TITLE
Update that supports newer version Redux-Saga

### PR DESCRIPTION
**Pull request by AugustoAleGon**
Update Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,8 +268,8 @@ it('works with unit tests', () => {
 
 ### Redux Saga v0.15
```
Comment (jfairbank) [inline README.md]: Maybe update this as well to something like "Redux Saga Support"

Comment (AugustoAleGon) [reply README.md]: Alright let me do it an PR

**Comment by AugustoAleGon** [comment]
Ready! That's better right?

**Comment by jfairbank** [comment]
Thanks!

**Comment by AugustoAleGon** [comment]
Your welcome. It is a pleasure to help great projects.

**Pull request merged**